schema/rdl_schema_0.1.json

```
@@ -93,44 +93,13 @@
 			"description": "Contact email"
 		},
   	"resources": {
+			"title": "Resources",
       "type": "array",
+			"description": "An array of one of more resource objects each describing a version of the risk dataset.",
```

To maintain an SSOT, don't state the type of fields and array items in descriptions:

```suggestion
			"description": "Information about the resources included in the dataset.",
```